From b1c99224dc0d318109832531c98ec24ef4c0020e Mon Sep 17 00:00:00 2001
From: tastytea <tastytea@tastytea.de>
Date: Wed, 23 Feb 2022 14:00:36 +0100
Subject: [PATCH] Link to webp and sass

We need to add the flags because we unbundle media-libs/libwebp and
dev-libs/libsass.
---
 vendor/github.com/bep/golibsass/internal/libsass/a__cgo.go | 2 +-
 vendor/github.com/bep/gowebp/internal/libwebp/a__cgo.go    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/vendor/github.com/bep/golibsass/internal/libsass/a__cgo.go b/vendor/github.com/bep/golibsass/internal/libsass/a__cgo.go
index af51b92..979d9e7 100644
--- a/vendor/github.com/bep/golibsass/internal/libsass/a__cgo.go
+++ b/vendor/github.com/bep/golibsass/internal/libsass/a__cgo.go
@@ -8,6 +8,6 @@ package libsass
 // #cgo CFLAGS: -O2 -fPIC
 // #cgo CPPFLAGS: -I../../libsass_src/include
 // #cgo CXXFLAGS: -g -std=c++0x -O2 -fPIC
-// #cgo LDFLAGS: -lstdc++ -lm
+// #cgo LDFLAGS: -lstdc++ -lm -lsass
 // #cgo darwin linux LDFLAGS: -ldl
 import "C"
diff --git a/vendor/github.com/bep/gowebp/internal/libwebp/a__cgo.go b/vendor/github.com/bep/gowebp/internal/libwebp/a__cgo.go
index 4b105ab..cec2faa 100644
--- a/vendor/github.com/bep/gowebp/internal/libwebp/a__cgo.go
+++ b/vendor/github.com/bep/gowebp/internal/libwebp/a__cgo.go
@@ -2,5 +2,5 @@
 
 package libwebp
 
-// #cgo linux LDFLAGS: -lm
+// #cgo linux LDFLAGS: -lm -lwebp
 import "C"
-- 
2.34.1

